sources/packages/web/src/views/student/StudentApplicationDetails.vue

```diff
@@ -133,25 +136,30 @@ export default {
         name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
       });
     };
-    const getProgramYear = async () => {
+    const getProgramYear = async (includeInActivePY?: boolean) => {
       programYear.value = await ApplicationService.shared.getProgramYearOfApplication(
         props.id,
+        includeInActivePY,
       );
     };
 
     const editApplicaion = async () => {
-      await getProgramYear();
-      router.push({
-        name: StudentRoutesConst.DYNAMIC_FINANCIAL_APP_FORM,
-        params: {
-          selectedForm: programYear.value.formName,
-          programYearId: programYear.value.programYearId,
-          id: props.id,
-        },
-      });
+      try {
+        await getProgramYear();
+        router.push({
+          name: StudentRoutesConst.DYNAMIC_FINANCIAL_APP_FORM,
+          params: {
+            selectedForm: programYear.value.formName,
+            programYearId: programYear.value.programYearId,
+            id: props.id,
+          },
+        });
+      } catch (error) {
+        toast.error("Program Year not active", "", TOAST_ERROR_DISPLAY_TIME);
```

Comment:
You can pass undefined instead of empty string for second argument. I know our composble has signature of string only but the toast.add allows undefined.  We can update the signature of composable.
![image](https://user-images.githubusercontent.com/54600590/147794800-e2f2b93b-2a4c-4a8d-87fe-630d20fd8726.png)

Reply:
![image](https://user-images.githubusercontent.com/77353155/148099501-befb0b0e-6b47-4634-abcb-cad0a6496cc4.png)